worker/src/queues/notify/workers/index.ts

```diff
@@ -16,4 +18,8 @@ export async function setupNotifyWorker() {
 
   logger.info({ PROCESS_QUEUE }, `starting 'notifyProcessHandler' on ${PROCESS_QUEUE} listeners`);
   await consumer.work(PROCESS_QUEUE, { newJobCheckInterval: 2000 }, notifyProcessHandler);
+
+  logger.info({ FAILURE_CHECK_QUEUE }, `starting 'notifyFailureHandler' on ${FAILURE_CHECK_QUEUE} listeners`);
+  await consumer.schedule(FAILURE_CHECK_QUEUE, `0 0 0 * *`);
+  await consumer.work(FAILURE_CHECK_QUEUE, { newJobCheckInterval: 43200000 }, notifyFailureHandler);
```

Comment:
Can you make this cron expression configurable, but default it to 9am? otherwise this would send an alert at midnight, but it's not that urgent. Also, it should be `*` for the day segment. At the moment it's run at midnight, on the 0th day (rather than * for every day).




It's also worth labelling `newJobCheckInterval` to however many hours it is, or use an expression